ctowasm/src/processor/symbolTable.ts

```diff
@@ -73,8 +73,10 @@ export class SymbolTable {
     } else {
       this.externalFunctions = {};
       this.parentTable = null;
-      this.dataSegmentByteStr = { value: "" };
-      this.dataSegmentOffset = { value: 0 };
+      // Initial value to reserve for the null space
+      this.dataSegmentByteStr = { value: '\\d0\\e0\\b0\\f0'  };
```

Comment:
any reason for this particular value? Although it shouldn't matter anyway

Reply:
We wanted a specific value in hex for debugging purposes. We chose "DEBF"  which stands for "Debugging Errors Before Failing".